.babelrc

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
@alfonsogarciacaro is fable using this? or should it go to the splitter config?

Reply:
Not Fable itself, but Babel will detect and use it: https://babeljs.io/docs/en/next/options#babelrc